ts/api/mock/messages.json

```diff
@@ -0,0 +1,44 @@
+[
```

Comment:
since we have PR #157 merged, this should not be needed anymore - @fpersico ?

Reply:
All mocks must be removed